src/main/java/com/wise23/chariteed/model/User.java

```diff
@@ -0,0 +1,165 @@
+package com.wise23.chariteed.model;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.validator.constraints.Length;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+import java.io.File;
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Collections;
+
+@Entity
+@Table(name = "users")
+public class User implements UserDetails {
+    @SequenceGenerator(name = "users_sequence", sequenceName = "users_sequence", allocationSize = 1)
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "users_sequence")
+    private Long id;
+
+    private String patientID;
+
+    private File file;
+
+    @NotNull(message = "First Name cannot be empty")
+    @Column(name = "first_name")
+    private String firstName;
+
+    @NotNull(message = "Last Name cannot be empty")
+    @Column(name = "last_name")
+    private String lastName;
+
+    @NotNull(message = "Email cannot be empty")
+    @Email(message = "Please enter a valid email address")
+    @Column(name = "email", unique = true)
+    private String email;
+
+    @NotNull(message = "Password cannot be empty")
+    @Length(min = 7, message = "Password should be atleast 7 characters long")
+    @Column(name = "password")
```

Comment:
This is a VERY, VERY BAD decision, to store authentification data together with the rest of data.

Reply:
passwords are **encrypted** to be stored in the database, why it is so bad?

Reply:
see https://github.com/Miao-99/charite-ed/pull/10#discussion_r1063579603